generators/app/templates/src/app/components/CustomText/styles.js

```diff
@@ -0,0 +1,54 @@
+import { StyleSheet } from 'react-native';
+
+import { blue, white, green, gray, black, transparent } from '../../../utils/colors';
+import { IS_SMALL_DEVICE } from '../../../utils/constants';
+
+const amountToResize = 2 * IS_SMALL_DEVICE;
+
+// Custom Text Style
```

Comment:
I'd write this as comments of the component, here may be more difficult to find.
```js
/**
 * You can add styles to Base like Family Font to be the Text styles base!
......
 */
export default class CustomText extends PureComponent {
```

Reply:
:+1: